users/miketronic/miketronic.c

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 @miketronic -- Mike B <mxb540@gmail.com>
+// SPDX-License-Identifier: GPL-2.0+
+
+#include "miketronic.h"
+
+// Add reconfigurable functions here, for keymap customization
+// This allows for a global, userspace functions, and continued
+// customization of the keymap.  Use _keymap instead of _user
+// functions in the keymaps
+
+
+// In keymaps, instead of writing _user functions, write _keymap functions
+// The __attribute__((weak)) allows for empty definitions here, and during
+// compilation, if these functions are defined elsewhere, they are written
+// over. This allows to include custom code from keymaps in the generic code
+// in this file.
+
+
+
+
+__attribute__ ((weak)) void matrix_init_keymap(void) { }
+__attribute__ ((weak)) void matrix_scan_keymap(void) { }
+__attribute__ ((weak)) void led_set_keymap(uint8_t usb_led) { }
+
+__attribute__ ((weak)) layer_state_t layer_state_set_keymap (layer_state_t state) {
+    return state;
+}
+
+
+// Init effect for RGB boards only
+// https://github.com/qmk/qmk_firmware/blob/master/docs/custom_quantum_functions.md
+void keyboard_post_init_user(void) {
+#ifdef RGB_MATRIX_ENABLE
+    rgblight_enable_noeeprom(); // enables Rgb, without saving settings
+    rgblight_sethsv_noeeprom(180, 255, 255); // sets the color to teal/cyan without saving
+    rgb_matrix_mode_noeeprom(RGB_MATRIX_SOLID_COLOR);
+#endif
+        matrix_init_keymap();
+}
+
+
+/*
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+#ifdef CONSOLE_ENABLE
+    uprintf("LAYER: %u\n", state);
+#endif
+    return state;
+}
+ */
```

Comment:
This should be uncommented, or it should be removed. 
eg: 
```suggestion
```
or
```suggestion
__attribute__ ((weak)) layer_state_t layer_state_set_keymap (layer_state_t state) {
    return state;
}

layer_state_t layer_state_set_user(layer_state_t state) {
    state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
    state = layer_state_set_keymap(state);
#ifdef CONSOLE_ENABLE
    uprintf("LAYER: %u\n", state);
#endif
    return state;
}

```

Reply:
Thanks, updated the _ADJUST to my _SYMB layer as an additional method to enable, also I don't have the _ADJUST layer.  I was keeping that code to use in the future.  I'll add "not yet implemented" work to an offline notebook going forward.